docs/how_to/composability.md

```diff
@@ -0,0 +1,23 @@
+# Composability
+
+
+GPT Index offers **composability** of your indices, meaning that you can build indices on top of other indices. This allows you to more effectively index your entire document tree in order to feed custom knowledge to GPT.
+
+Composability allows you to to define lower-level indices for each document, and higher-order indices over a collection of documents. To see how this works, imagine defining 1) a tree index for the text within each document, and 2) a list index over each tree index (one document) within your collection.
+
+![](/_static/composability/diagram.png)
+
+During a query, we would start with the top-level list index. Each node in the list corresponds to an underlying tree index. 
+
+![](/_static/composability/diagram_q1.png)
```

Comment:
Thank you for putting up the diagrams. They make this much easier to understand!